### DISCUSSION TITLE
`vendor-xxxxx.js` loading slowly for Seq hosted on Windows servers

### DISCUSSION THREAD

**Discussion by larenelg** [datalust/seq-tickets#1081] [Help]
**Describe the bug**
`vendor-xxxxx.js` taking longer than 30 seconds to load for customers with Seq hosted on Azure Windows VMs.

**To Reproduce**
1. Spin up a new instance of Seq on an Azure Windows VM.
2. Navigate to your Seq url.
3. Open browser developer tools.
4. See `vendor-xxxxx.js` loading for at least 30 seconds (~800kb)

**Expected behavior**
Even though the size of vendor-xxxxx.js is relatively large, it should only take 1-2 seconds to load, at the most.

**Screenshots**
![image](https://user-images.githubusercontent.com/7476736/102304395-189e9100-3fa9-11eb-895c-e9018fa83845.png)

**Environment (please complete the following information):**
 - OS: Windows
 - Browser All
 - Seq Version 2020.3, 2020.4
 - Using Docker? No

**Additional context**
Problem seems to be related to using `http.sys` and using Response Compression in ASP.NET Core.
https://github.com/dotnet/aspnetcore/issues/14455

**Comment by nblumhardt** [comment]
We're converting this to a discussion to track until we get an update on the upstream ticket.

**Reply by n1njab0b** [reply]
I'm having this exact problem with the current (2023.1.9229) release, inexplicably slow download of vendor-xxxxx.js in our browsers.  Any workarounds for this?  Perhaps somehow disabling response compression?

**Reply by nblumhardt** [reply]
Hi @n1njab0b - we've actually made a small configuration change in later 2023.x builds that may improve this. Do you still observe the issue after upgrading to 2023.3.9691?

**Reply by n1njab0b** [reply]
Following up on this to let you know that the problem is resolved on the latest update. Thanks again!

**Reply by nblumhardt** [reply]
Excellent! Thanks for closing the loop 👍

**Comment by n1njab0b** [comment]
Glad to hear! I’ll let you know when I get a maintenance window to update. On Aug 1, 2023, at 6:12 PM, Nicholas Blumhardt ***@***.***> wrote:﻿
Hi @n1njab0b - we've actually made a small configuration change in later 2023.x builds that may improve this. Do you still observe the issue after upgrading to 2023.3.9691?
